ui/src/components/pamFileList/index.vue

```diff
@@ -417,19 +435,32 @@ const handleBeforeLeave = (tabName: string) => {
     return true;
   }
 
-  settingDrawer.value = false;
+  if (tabName === 'fileManage') {
+    settingDrawer.value = false;
+
+    nextTick(() => {
+      const drawerRef: HTMLElement = document.getElementsByClassName('n-drawer')[0] as HTMLElement;
 
-  return true;
+      if (drawerRef) {
+        drawerRef.style.width = '700px';
+      }
+    });
+
+    return true;
+  }
 };
 
 const columns = createColumns();
 
 onMounted(() => {
+  useFileManage();
   mittBus.on('open-fileList', handleOpenFileList);
+  mittBus.on('open-setting', handleOpenSetting);
 });
 
 onBeforeUnmount(() => {
   mittBus.off('open-fileList', handleOpenFileList);
+  mittBus.off('open-setting', handleOpenSetting);
 });
 </script>
 
```

Comment:
There is no issue with the current code, but some small improvements could be made based on best practices:
```yaml
# 1. Add `@title` attribute to <NButton> components for better accessibility.
#
<style scoped lang="scss">
  @import "naive-ui/vars/css-vars";
</style>

<script setup>
useHead({
  script: ['~/public/script.js']
});
usingComponents([
  ...
]);

...
```

This change will improve accessibility by adding screen reader support for the title of buttons.

To enhance performance and efficiency further:

* Update all instances of `<i>` tags to have proper inline styling (e.g., using CSS properties instead of HTML entities) in components like `/views/components/i.vue`.

Update each file in this project so that it follows the latest standard naming conventions (CSS variables, consistent component files). This improves SEO as search engines can now accurately index these files.

For more specific optimizations, consult other libraries' documentation, such as Naive UI's [documentation](https://naiveui.com/) and Carbon Icons' [documentation](https://carbon.notion.site/).

Please note: These changes require modifications across your entire project structure due to its complexity. Therefore, they should only be applied when making significant revisions.

These were just general improvements, while optimizing an exact piece of JavaScript code involves looking at a broader context including different aspects of the application.